### ISSUE TITLE
Need to add photo and video a gallery page

### ISSUE THREAD

**Issue by xxxjude** [acf-sack/sack-site#378]
Can we have a gallery page to display photos and also upload demo videos?

1. The photos taken at the Launching of Digital Learning program on 8-Jan. Especially the Sponsoring Mobile Tabs to 3 students.
2. Need to upload demo videos done for Google Classroom.  

Also, where can i upload videos and pictures for your reference?

**Comment by kumuditha-udayanga** [comment]
We can upload the videos to the youtube channel and embed them on the website. WDYT @Piumal1999 @jayasanka-sack?

**Comment by Yoshitha-SACK** [comment]
Shall I work on this ?

**Comment by Piumal1999** [comment]
> We can upload the videos to the youtube channel and embed them on the website. WDYT @Piumal1999 @jayasanka-sack?

Yeah, that's the best approach for creating the video gallery. For photos, are we going to directly upload the content to the GitHub repository?
